06-building-a-program-start-to-finish/solving-the-problem/output/inverted.js

```diff
@@ -0,0 +1,209 @@
+/**
```

Comment:
This file name is not proper. Please meet with your peers on what is preferred.

Also review your parameter/variable names in this entire file. 

You multiple cases of eslint "_**exceeds the maximum line length of 80**_" violations on this file. Please them all